lib/ModulesPerl6/DbBuilder/Dist.pm

```diff
@@ -1,6 +1,9 @@
 package ModulesPerl6::DbBuilder::Dist;
 
 use ModulesPerl6::DbBuilder::Log;
+use ModulesPerl6::DbBuilder::Dist::Source::GitHub;
```

Comment:
How come this `use` was added BTW? Doesn't Module::Pluggable already load it?

Reply:
Sorry, I forgot to delete this line.

Reply:
Fixed by #104